### DISCUSSION TITLE
Using multiple .md files for different pages generation

### DISCUSSION THREAD

**Discussion by vazgmik** [ankitskvmdam/clean-jsdoc-theme#251] [General]
Hello, can you explain how can I use multiple .md files for showing different pages in theme? For example, I need to add "User manual" menu and try to link it with user_manual.md file in project to generate manual page. Is it possible?

**Comment by ankitskvmdam** [comment]
v5 supports a docs directory (`opts.docs`) for multiple Markdown pages, each with its own sidebar group. Guide: https://ankdev.me/clean-jsdoc-theme/guides/build-a-guides-site/ Closing as resolved.
